app/presenters/sufia/file_set_presenter.rb

```diff
@@ -36,6 +36,10 @@ def itemtype
       'http://schema.org/CreativeWork'
     end
 
+    def processing?
+      # TODO: Add Status to FileSet to track the staus of processing
```

Comment:
Can you tweak this comment to be:

`# TODO: Refactor this away per https://github.com/projecthydra/sufia/pull/1592`

:question: